lib/robots/dor_repo/gis_delivery/load_vector.rb

```diff
@@ -42,50 +32,34 @@ def perform_work
             # sniff out shapefile from extraction
             Dir.chdir(tmpdir)
             shpfn = Dir.glob('*.shp').first
+
             sqlfn = shpfn.gsub(/\.shp$/, '.sql')
             errfn = 'shp2pgsql.err'
             logger.debug "load-vector: #{bare_druid} is working on Shapefile: #{shpfn}"
 
             # first try decoding with UTF-8 and if that fails use LATIN1
             begin
-              run_shp2pgsql(projection, 'UTF-8', shpfn, schema, sqlfn, errfn)
+              normalizer.run_shp2pgsql('4326', 'UTF-8', shpfn, schema, sqlfn, errfn)
             rescue RuntimeError
-              run_shp2pgsql(projection, 'LATIN1', shpfn, schema, sqlfn, errfn)
+              normalizer.run_shp2pgsql('4326', 'LATIN1', shpfn, schema, sqlfn, errfn)
             end
```

Comment:
had to explicitly reference the normalizer class here, else it was looking for `run_shp2pgsql` in this class instead of the normalizer class (perhaps because we are in a begin/end block?)